scripts/sign-release-images.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"";
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+: "${GIT_HASH:?Environment variable empty or not defined.}"
+: "${GIT_VERSION:?Environment variable empty or not defined.}"
+
+if [[ ! -f $ARTIFACT ]]; then
+    echo "$ARTIFACT not found"
+    exit 0
```

Comment:
do we need to fail if the ARTIFACT isn't provided?

Reply:
opssy :)

Reply:
thanks for the catch! i was doing some tests and forgot to revert